engine/proto/master.proto

```diff
@@ -239,17 +239,15 @@ message Job {
 
 message CreateJobRequest {
     Job job = 1;
-    string tenant_id = 2;
-    string project_id = 3;
+    ProjectInfo project_info = 2;
```

Comment:
Here is intended to not use `ProjectInfo`, since it will make query parameters more complicated.

Reply:
Ok. Besides, maybe we should remove ProjectInfo to keep proto in a uniform form.